src/webgpu/api/validation/queue/copyToTexture/CopyExternalImageToTexture.spec.ts

```diff
@@ -187,11 +183,14 @@ export const g = makeTestGroup(CopyExternalImageToTextureTest);
 g.test('source_canvas,contexts')
   .desc(
     `
-  Test HTMLCanvasElement as source image with different contexts.
+  Test HTMLCanvasElement and OffscreenCanvas as source image with different contexts.
```

Comment:
The test doesn't appear to test OffscreenCanvas..?

Reply:
Oh, yes, tried to merge them but think better to split them due to the different valid `getContext` param type.